File: cache/cache-annotation-processor/src/test/java/ru/tinkoff/kora/cache/annotation/processor/CacheRunner.java
package ru.tinkoff.kora.cache.annotation.processor;

import org.jetbrains.annotations.Nullable;
import ru.tinkoff.kora.cache.caffeine.CaffeineCacheConfig;

import java.time.Duration;

final class CacheRunner {

    private CacheRunner() { }

    public static CaffeineCacheConfig getConfig() {
        return new CaffeineCacheConfig() {
            @Nullable
            @Override
            public Duration expireAfterWrite() {
                return null;
            }

            @Nullable
            @Override
            public Duration expireAfterAccess() {
                return null;
            }

            @Nullable
            @Override
            public Integer initialSize() {
                return null;
            }
        };
    }
}

File: cache/cache-symbol-processor/src/test/kotlin/ru/tinkoff/kora/cache/symbol/processor/CacheRunner.kt
package ru.tinkoff.kora.cache.symbol.processor

import ru.tinkoff.kora.cache.caffeine.CaffeineCacheConfig
import java.time.Duration

class CacheRunner {

    companion object {

        fun getConfig() : CaffeineCacheConfig {
            return object : CaffeineCacheConfig {
                override fun expireAfterWrite(): Duration? {
                    return null;
                }

                override fun expireAfterAccess(): Duration? {
                    return null;
                }

                override fun initialSize(): Int? {
                    return null;
                }
            }
        }
    }
}
